client/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Button',
+};
+export const Default = () => (
+  <Button variant="transparent"> Button </Button>
+);
+export const Primary = () => (
+  <Button variant="blue">  Button </Button>
+);
+
+export const Light = () => (
+  <Button variant="light"> Button </Button>
+);
+
+export const Success = () => (
+  <Button variant="success">Button</Button>
+);
+
+export const Danger = () => (
+  <Button variant="danger">Button</Button>
+);
+
+export const Yellow = () => (
+  <Button variant="yellow"> Button </Button>
+);
+
+export const Grey = () => (
+  <Button variant="grey"> Button </Button>
+);
+
+export const Bouncy = () => (
+  <Button variant="bouncy"> Button </Button>
+);
+
+export const WithIcon = () => (
+  <Button variant="with-icon"> Button </Button>
```

Comment:
Typo here. The variant is called `with--icon`. 

Also, you should add an icon to this, to have a real example. If you wanna see where this button is used and how it's supposed to look like, you can look into `SocialMediaButtons.tsx`.

PS: you may need to add `withIcon` as a prop instead of as a `variant` since we can have **blue**, **success**, **danger** buttons with variants.